doc/stdlib/libc.rst

```diff
@@ -40,7 +40,10 @@
 
 .. function:: time(t::TmStruct)
 
-   Converts a ``TmStruct`` struct to a number of seconds since the epoch.
```

Comment:
Splitting things up is a good idea, I can give that a go.

I think that if we're seriously considering adding more syntaxes like this, #11850 becomes much more important.